src/services/ondemand-channel/api/register.ts

```diff
@@ -628,6 +633,7 @@ const attemptChannelOpen = async ({
         privateChannel,
         spendUnconfirmed,
         false,
+        taprootChannel,
```

Comment:
I think we should set this to `false` here, otherwise users on Blixt <0.6.8 won't be able to open channels.
At least for some months, then we can probably simplify this part in general